akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowLogSpec.scala

```diff
@@ -0,0 +1,212 @@
+/**
+ * Copyright (C) 2014-2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream.scaladsl
+
+import akka.event.{ DummyClassForStringSources, Logging }
+import akka.stream._
+import akka.stream.scaladsl.OperationAttributes.LogLevels
+import akka.stream.testkit.{ AkkaSpec, ScriptedTest }
+import akka.testkit.TestProbe
+
+import scala.concurrent.forkjoin.ThreadLocalRandom.{ current ⇒ random }
+import scala.util.control.NoStackTrace
+
+class FlowLogSpec extends AkkaSpec("akka.loglevel = DEBUG") with ScriptedTest {
+
+  implicit val mat: FlowMaterializer = ActorFlowMaterializer()
+
+  def logProbe(): TestProbe = {
+    val p = TestProbe()
+    system.eventStream.subscribe(p.ref, classOf[Logging.LogEvent])
+    p
+  }
```

Comment:
no good reason, you're right val may be better here (could expose if more msgs sent than I expected)